### DISCUSSION TITLE
Timeout in 2FA

### DISCUSSION THREAD

**Discussion by jvegh** [community/community#134792] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Maybe due to my  provider(s), in the 95% of cases, I receive a timeout error when attempting to use 2FA. Is it possible to adjust the time limit?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @jvegh ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ebndev** [comment]
Hey there @jvegh,

Thanks so much for reaching out in GitHub's Community! I'm not sure what your issue may be if it's with your provider, but you might want to consider setting up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

We put together a [FAQ](https://github.com/orgs/community/discussions/68240) in case you have any additional questions that may be answered there!
